simpleflow/swf/executor.py

```diff
@@ -939,6 +939,21 @@ def replay(self, decision_response, decref_workflow=True):
                 self.decref_workflow()
             if self._append_timer:
                 self._add_start_timer_decision('_simpleflow_wake_up_timer')
+
+            if not self._decisions_and_context.execution_context:
```

Comment:
Not sure I understand this part. Couldn't we just send an empty string no matter what was the last decision context? Pros: super easier, cons: ?? maybe some noise in the history/webflow?

Reply:
Yes we can. But I was so proud of `last` 🙂

Reply:
I can understand definitely ;) maybe just extract it in a private method to reduce the noise in this already too long replay() method?